src/me/components/PinnedItems.tsx

```diff
@@ -72,34 +79,36 @@ const PinnedItems: FC = () => {
               key={item.id}
               testID="pinneditems--card"
               className="pinned-items--card"
+              contextMenuInteraction="showOnHover"
               contextMenu={
-                <Context>
-                  <Context.Menu
+                <FlexBox margin={ComponentSize.ExtraSmall}>
+                  <ConfirmationButton
+                    color={ComponentColor.Colorless}
                     icon={IconFont.Trash_New}
-                    color={ComponentColor.Danger}
-                    testID="pinneditems-delete--menu"
-                  >
-                    <Context.Item
-                      label="Unpin"
-                      action={() => handleDeletePinnedItem(item.id)}
-                      testID="pinneditems-delete--confirm"
-                    />
-                  </Context.Menu>
-                </Context>
+                    shape={ButtonShape.Square}
+                    size={ComponentSize.ExtraSmall}
+                    confirmationLabel="Unpin this item?"
+                    onConfirm={() => handleDeletePinnedItem(item.id)}
+                    confirmationButtonText="Yes"
+                    testID="context-delete-menu"
+                  />
+                </FlexBox>
               }
+              style={{marginTop: '0px'}}
             >
-              <ResourceCard.Name
-                testID="pinneditems--type"
-                name={capitalize(item.type)}
-              />
+              <ResourceCard.Meta>
+                <Heading
+                  type={Typeface.IBMPlexMono}
+                  element={HeadingElement.H5}
+                >
+                  {item.type.toUpperCase()}
```

Comment:
Correct, it would! except it's for the headers, the `DASHBOARD` and `TASK`, not the name of the resource

<img width="1236" alt="Capture d’écran, le 2021-11-11 à 12 13 25 PM" src="https://user-images.githubusercontent.com/18511823/141362793-e75f8a84-13f4-4397-99ac-3a28346f93c6.png">
.